### ISSUE TITLE
Need to visualize forecast results

### ISSUE THREAD

**Issue by monkeycc** [ultralytics/ultralytics#720]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar feature requests.

### Description

Need to visualize forecast results

GUI needs to get the result image

### Use case

```
yolo.det.visualize
yolo.seg.visualize(img, result)

yolo.seg.visualize
yolo.seg.visualize(img, result)
```

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by AyushExel** [comment]
You can do this
```
from ulltralytics import YOLO

model = YOLO("yolov8n.pt") 
model.predict(img, show=True)
```

**Comment by monkeycc** [comment]
I need to read the picture
PySimpleGUI  GUI to call images
Opencv saving pictures

show=True
Can't meet the demand
Not picture data

@AyushExel

**Comment by AyushExel** [comment]
Oh I see. You can plot the images using the bbox data from results. 
Results = model(img) 
See usage here https://docs.ultralytics.com/predict/

**Comment by monkeycc** [comment]
Want to add visual prediction result image

```
Results = model(img)

img2 = yolo.cls.visualize(img, Results)
img2 = yolo.det.visualize(img, Results)
img2 = yolo.seg.visualize(img, Results)
```

In this way, users do not need to implement picture frames

**Comment by AyushExel** [comment]
Yes, we don't have this yet. I plan to work on something similar this week

**Comment by bronfo1** [comment]
who has the spare time and interest about object recognition and detecting, you can contact me: okplay@gmail.com.

**Comment by bronfo1** [comment]
@monkeycc hi monkeycc

**Comment by glenn-jocher** [comment]
@monkeycc this issue should now be resolved in #747
